### PR TITLE
Added mat files for NAV benchmark and finished creating readme table as discussed

### PR DESCRIPTION
**Pull request by toladnertum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,24 @@ Event info: https://cps-vo.org/group/ARCH/FriendlyCompetition
...
-| [NAV](./benchmarks/NAV) | robust | default | `TODO` | new | 
-| [QUAD](./benchmarks/QUAD) | `TODO` | default | dim 3 over time | - | 
-| [Single Pendulum](./benchmarks/Single_Pendulum) | `TODO` | default | `TODO` | modified | 
-| [Tora Heterogeneous](./benchmarks/Tora_Heterogeneous) | `TODO` | default | `TODO` | - | 
-| [VCAS](./benchmarks/VCAS) | `TODO` | default | `TODO` | - | 
+| Benchmark | Instance | Specification | Network | Visualization | Comment |
+|-----------|----------|---------------|-----------|---------------|---------|
+| [ACC](./benchmarks/ACC) | safe-distance | default | relu | distance over time | - | 
+| [Airplane](./benchmarks/Airplane) | continuous | t\in[0,20] | relu | [2,7] | modified spec | 
+| [Airplane](./benchmarks/Airplane) | discrete | t\in{0,1,...,20} | relu | [2,7] | modified spec |
```
Comment (schillic) [inline README.md:15]: @toladnertum: I have not heard about a discrete version of the Airplane model. Where is this described?

Comment (toladnertum) [reply README.md:15]: It says so in last year's report (end of Sec. 3.7 in the "Specification" paragraph). I wasn't aware of it as well (and I think nobody considered it last year).
image

Comment (schillic) [reply README.md:15]: I see, thanks for the reference. There are also discrete-time variants for the two pendulum models. They come from a time when some tools could only do discrete-time analysis, and this was considered an alternative. I do not have an opinion whether or not to include them in the table (and the report), but if included, they should probably all be included consistently.

Comment (toladnertum) [reply README.md:15]: I guess if they are not really considered anymore, we can also drop them from the table. Maybe we just add a note below saying that if someone uses the discrete variant of a benchmark, we add a suffix to the instance name, e.g., `reach-discrete`.

**Pull request merged**